docs/content/guides/traffic_management/hello_world/_index.md

```diff
@@ -18,73 +18,88 @@ If there are no routes configured, Envoy will not be listening on the gateway po
 
 ---
 
-## Preparing the Environment
+## Before you begin
 
-To follow along in this guide, you will need to fulfill a few prerequisites.  
+1. Create a [Kubernetes cluster]({{< versioned_link_path fromRoot="/installation/platform_configuration/cluster_setup/" >}}).
 
-### Prerequisite Software
+2. [Install Gloo Gateway]({{< versioned_link_path fromRoot="/installation/gateway/kubernetes" >}}).
 
-Your local system should have `kubectl` and `glooctl` installed, and you should have access to a Kubernetes deployment to install Gloo Gateway.
+3. Make sure that you have the following command line tools installed.
+   
+   * [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+   * `glooctl`
 
-* [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-* `glooctl`
-* Kubernetes v1.11.3+ deployed somewhere. [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) is a
-great way to get a cluster up quickly.
+---
+
+## Enable service discovery {#service-discovery}
```

Comment:
added a step to the sample app guide to enable discovery so that the rest of it works